src/main/java/de/terrestris/shogun/dao/DatabaseDao.java

```diff
@@ -83,6 +81,9 @@
 import de.terrestris.shogun.model.Group;
 import de.terrestris.shogun.model.MapLayer;
 import de.terrestris.shogun.model.User;
+import de.terrestris.shogun.service.UserAdministrationService;
```

Comment:
Eclipse imports these dynamically also when {@link UserAdministrationService} is used, so javadoc can resolve links.